subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/IsolatedAntBuilderMemoryLeakIntegrationTest.groovy

```diff
@@ -82,8 +83,10 @@ class IsolatedAntBuilderMemoryLeakIntegrationTest extends AbstractIntegrationSpe
         """
     }
 
-    void 'CodeNarc does not fail with PermGen space error'() {
+    @Issue('https://github.com/gradle/gradle/issues/22172')
```

Comment:
Seems odd we're marking this with `@Issue`. Usually we use this annotation to mark regression tests for a particular issue, not issues with the test itself. IMO a TODO is more fitting. Applies below as well